sdks/python/tox.ini

```diff
@@ -86,19 +86,35 @@ commands =
   python setup.py test
 passenv = TRAVIS*
 
-[testenv:lint]
+[testenv:lint1]
 deps=
   nose==1.3.7
   pycodestyle==2.3.1
   pylint==1.7.2
   future==0.16.0
   isort==4.2.15
+  flake8==3.5.0
 whitelist_externals=time
 commands =
   time pip install -e .[test]
   time {toxinidir}/run_pylint.sh
 passenv = TRAVIS*
 
+[testenv:lint2]
```

Comment:
perhaps rename this to `lint3` `lint_py3` or something similar. 2 might be confusing.